platform/broadcom/sonic-platform-modules-inventec/debian/platform-modules-d6332.install

```diff
@@ -0,0 +1,3 @@
+d6332/utils/inventec_d6332_util.py              /usr/local/bin
+systemd/platform-modules-d6332.service          /lib/systemd/system
+d6332/utils/sonic_platform-1.0-py2-none-any.whl /usr/share/sonic/device/x86_64-inventec_d6332-r0
```

Comment:
We are in the process of migrating SONiC to Python 3. During the transition, we would like all vendors to build, install and copy both a Python 2 sonic_platform package, and a Python 3 package (sonic_platform-1.0-py3-none-any.whl). Can you please also take care of this?

Reply:
hi @jleveque ,
I've tried to build Python3 platform API package, and install the new image on our DUT . However, I've encountered some problems when I tried to test it inside the pmon docker (please refer to the log below)

```
root@sonic:/usr/local/bin# python3
Python 3.7.3 (default, Jul 25 2020, 13:03:44)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sonic_platform.psu import Psu
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sonic_platform_base/sonic_sfp/sfputilhelper.py", line 17, in <module>
    from portconfig import get_port_config
ModuleNotFoundError: No module named 'portconfig'


During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/sonic_platform/qsfp.py", line 15, in <module>
    from sonic_platform_base.sonic_sfp.sfputilhelper import SfpUtilHelper
  File "/usr/local/lib/python3.7/dist-packages/sonic_platform_base/sonic_sfp/sfputilhelper.py", line 22, in <module>
    raise ImportError("%s - required module not found" % str(e))
ImportError: No module named 'portconfig' - required module not found

During handling of the above exception, another exception occurred:
......
```
I'm not sure it's caused by some build errors or etc, so python can't find the required module, or maybe some modules are not adapted to python3 yet...?

And when I check outside the docker, it seems that the sonic_platform module isn't installed successfully under pip3, but the python2 module works well. (please refer to the log below)
```
root@sonic:~# pip3 list | grep platform
sonic-platform-common 1.0
root@sonic:~# pip list | grep platform
sonic-platform               1.0
sonic-platform-common        1.0
root@sonic:~# cd /usr/share/sonic/device/x86_64-inventec_d6332-r0/
root@sonic:/usr/share/sonic/device/x86_64-inventec_d6332-r0# ls
board_thermal_1  coretemp            installer.conf     pmon_daemon_control.json             sonic_platform-1.0-py3-none-any.whl
board_thermal_2  custom_init.soc.j2  INVENTEC-D6332     psu1
board_thermal_3  custom_led.bin      led_proc_init.soc  sensors.conf
board_thermal_4  default_sku         plugins            sonic_platform-1.0-py2-none-any.whl
root@sonic:/usr/share/sonic/device/x86_64-inventec_d6332-r0#
```
I'm not sure if there's something I missed after adding sonic_platform-1.0-py3-none-any.whl to platform-modules-d6356.install? (as the log shows, the .whl file is under the correct path but not installed)

Would you please give me some advice about these issues? Thank you very much for your help!

Reply:
@CynthiaINV: The first problem you stated would be expected if the Python 3 version of the `sonic-config-engine` package is not installed in the container, as it is currently a dependency. Support for Python 3 was added to this package very recently, so it may not be present in your image. However, this is OK for now, since we haven't yet made the PMon daemons Python 3-compatible, so the package is not expected to get used just yet. As long as both the Python 2 and Python 3 wheels are getting installed in the PMon container, which it looks like they are, I think you're good for now.

The second issue looks like an actual problem, as it appears the Python 3 wheel is not getting installed in the host OS. You probably need to look at the image build logs to see why it's not getting installed. Please let me know if you have further questions.

Reply:
Hi @jleveque,
Thank you very much for your advice! I'll try to solve the second issue.

Reply:
hi @jleveque ,
I've tried to solve the sonic_platform not installed on linux issue, it can be installed with pip3 now. ( please refer to the log below)
```
root@sonic:/home/admin# pip3 list | grep platform
sonic-platform        1.0
sonic-platform-common 1.0
root@sonic:/home/admin# pip list | grep platform
sonic-platform               1.0
sonic-platform-common        1.0
root@sonic:/home/admin#
```
however, as the portconfig module can't installed issue mentioned above, the python3 version of platform API is still unested.
Thank you very much for your help!